frontend/src/metabase/.eslintrc

```diff
@@ -25,6 +25,10 @@
           {
             "name": "moment-timezone",
             "message": "Moment is deprecated, please use dayjs"
+          },
+          {
+            "name": "@storybook/testing-library",
```

Comment:
I believe we should make this (and other rules) to be in sync with the one in enterprise folder as well. I tested that I can still import `@storybook/testing-library` inside the enterprise folder

https://github.com/metabase/metabase/blob/8675cdc1045ac8773aed0d9ad53b9bc00a02461f/enterprise/frontend/.eslintrc#L7-L16

Reply:
https://github.com/metabase/metabase/issues/47083